src/commcare_cloud/ansible/roles/bootstrap-users/defaults/main.yml

```diff
@@ -3,3 +3,4 @@ bash_PS1: |
   '\[\e]0;({{ env_monitoring_id }}) \u@{{ alt_hostname|default('\h') }}: \w\a\]\[\033[01;33m\]({{ env_monitoring_id }})\[\033[00m\] \[\033[01;32m\]\u@{{ alt_hostname|default('\h') }}\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
 cchq_uid: null
 cchq_gid: null
+ssh_user: ansible
```

Comment:
@dannyroberts Need some suggestions here. Could you please help? 
I ran out of ideas and mentioned default user as `ansible` and skipping task if it ssh_user value 'ansible'. I was facing error 'user (variable) is not defined'

Reply:
Can you explain some more? I don't think I'm following

Reply:
this worked when I used `ssh_user is not none` in the playbook. Tested and pushed changes.